src/main/java/com/example/raiseTechcoursetask10/entity/Skiresort.java

```diff
@@ -0,0 +1,44 @@
+package com.example.raiseTechcoursetask10.entity;
```

Comment:
Javaではパッケージ名は小文字のみを使います。
```suggestion
package com.example.raisetechcoursetask10.entity;
```

Reply:
あーーーーー、そうでした。
すみません！！